### PR TITLE
Allow Symfony 6.0

### PR DESCRIPTION
**Pull request by franmomu**
| Q | A
| --- | ---
| Branch? | 2.x
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | 
| License | MIT
| Doc |

### CODE REVIEW

File: .github/workflows/phpunit.yml [hunk 1]
```diff
@@ -73,6 +79,11 @@ jobs:
           composer global require --no-interaction --no-progress symfony/flex:^1.11
           composer config extra.symfony.require ${{ matrix.symfony }}
 
+      - name: Remove non-compatible dependencies with Symfony 6
+        if: matrix.symfony == '6.0.*'
+        run: |
+          composer remove enqueue/enqueue-bundle --dev --no-update
```
Comment (dbu) [inline .github/workflows/phpunit.yml:85]: that is fine

**Review by dbu — APPROVED** [review]
awesome, thanks!

**Comment by dbu** [comment]
also merged it up into 3.x

**Comment by dbu** [comment]
i will wait with tagging the release until symfony 6 is out, just in case.

**Comment by mbabker** [comment]
> i will wait with tagging the release until symfony 6 is out, just in case.

SF6 is out, can this be released please?

**Comment by dbu** [comment]
right. here you go: https://github.com/liip/LiipImagineBundle/releases/tag/2.7.3

**Pull request merged**